packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -148,7 +153,20 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
     if (onToggleCollapse) {
       onToggleCollapse(group!);
     }
+  };
+
+  private _onKeyUp = (ev: React.KeyboardEvent<HTMLElement>): void => {
+    const shouldOpen = this.state.isCollapsed && ev.which === getRTLSafeKeyCode(KeyCodes.right);
```

Comment:
This should pass theme to handle contextual RTL.

Reply:
is that not what `getRTLSafeKeyCode` does?

Reply:
```
export function getRTLSafeKeyCode(key: number, theme: { rtl?: boolean } = {}): number {
  if (getRTL(theme)) {
    if (key === KeyCodes.left) {
      key = KeyCodes.right;
    } else if (key === KeyCodes.right) {
      key = KeyCodes.left;
    }
  }

  return key;
}
```

Reply:
Yes, it will use theme if passed. However, this call to `getRTLSafeKeyCode` is not passing theme, so it will fall back to global detection.

Reply:
Ok that makes sense. it won't look at contextual theme unless it is passed in. Can do

Reply:
pushed